mlflow/pyfunc/__init__.py

```diff
@@ -925,6 +925,15 @@ def __repr__(self):
             info["flavor"] = self._model_meta.flavors[FLAVOR_NAME]["loader_module"]
         return yaml.safe_dump({"mlflow.pyfunc.loaded_model": info}, default_flow_style=False)
 
+    @experimental
+    def get_raw_model(self):
+        """
+        Get the underlying raw model if the model wrapper implemented `get_raw_model` function.
+        """
+        if hasattr(self._model_impl, "get_raw_model"):
```

Comment:
can we update the wrappers so they all have the `model` attribute?

Reply:
Can we instead update all wrappers to have `get_raw_model`? Adding `self.model=...` seems redundant, and we can't directly replace the attribute name either.

Reply:
Q:

Not all models have `get_raw_model` method.
Shall we add  `get_raw_model` for all kinds of models ?

Reply:
Yes that's the proposal